docs/site/express-with-lb4-rest-tutorial.md

```diff
@@ -0,0 +1,304 @@
+    hello-world: A simple hello-world Application using LoopBack 4.
+    log-extension: An example extension project for LoopBack 4.
+    rpc-server: A basic RPC server using a made-up protocol.
+    > express-composition: A simple Express application that uses LoopBack 4 REST API.
+    ```
+
+2.  Switch to the directory.
+
+    ```sh
+    cd loopback4-example-express-composition
+    ```
+
+3.  Finally, start the application!
+
+    ```sh
+    $ npm start
+
+    Server is running at http://127.0.0.1:3000
+    ```
+
+### Create your LoopBack Application
+
+#### Scaffold your Application
+
+Run `lb4 app note` to scaffold your application and fill out the following
+prompts as follows:
+
+```sh
+$ lb4 app
+? Project description: An application for recording notes.
+? Project root directory: (note)
+? Application class name: (NoteApplication)
+❯◉ Enable tslint: add a linter with pre-configured lint rules
+ ◉ Enable prettier: add new npm scripts to facilitate consistent code formatting
+ ◉ Enable mocha: install mocha to assist with running tests
+ ◉ Enable loopbackBuild: use @loopback/build helpers (e.g. lb-tslint)
+ ◉ Enable vscode: add VSCode config files
+ ◉ Enable repositories: include repository imports and RepositoryMixin
+ ◉ Enable services: include service-proxy imports and ServiceMixin
+ # npm will install dependencies now
+ Application note was created in note.
+```
+
+#### Add Note Model
+
+Inside the project folder, run `lb4 model` to create the `Note` model with
+`Entity` model base class. Add an `id` property with type `number`, a required
+`title` property with type `string`, and a `content` property of type `string`.
+
+#### Add a DataSource
+
+Now, let's create a simple in-memory datasource by running the
+`lb4 datasource ds` command and the following full path to file:
+`./data/ds.json`.
+
+Similar to the `Todo` example, let's create the `ds.json` by creating a data
+folder at the application's root.
+
+```sh
+$ mkdir data
+$ touch data/ds.json
+```
+
+Then copy and paste the following into the `ds.json` file:
+
+```json
+{
+  "ids": {
+    "Note": 3
+  },
+  "models": {
+    "Note": {
+      "1": "{\"title\":\"Things I need to buy\",\"content\":\"milk, cereal, and waffles\",\"id\":1}",
+      "2": "{\"title\":\"Great frameworks\",\"content\":\"LoopBack is a great framework\",\"id\":2}"
+    }
+  }
+}
+```
+
+#### Add Note Repository
+
+To create the repository, run the `lb4 repository` command and choose the
+`DsDataSource` as the datasource and `Note` model as the model.
+
+#### Add Note Controller
+
+To complete the `Note` application, create a controller using the
+`lb4 controller note` command, with the `REST Controller with CRUD functions`
+type, `Note` model, and `NoteRepository` repository. The `id`'s type will be
+`number` and base HTTP path name is the default `/notes`.
+
+### Create a Facade Express Application
+
+Create a new file **src/server.ts** to create your Express class:
```

Comment:
Nitpick: since we show installing dependencies first, can we mention `Let's install dependencies for express module and create a new file...` here?